sdk/eventgrid/azure-messaging-eventgrid/src/samples/java/com/azure/messaging/eventgrid/samples/EventGridEventJavaDocCodeSnippet.java

```diff
@@ -0,0 +1,179 @@
+
+        // Send a list of EventGridEvents to the EventGrid service altogether.
+        // This has better performance than sending one by one.
+        eventGridEventPublisherClient.sendEvents(new ArrayList<EventGridEvent>() {
+            {
+                add(eventGridEvent);
+                // add more EventGridEvents objects
+            }
+        }).block();
+        // END: com.azure.messaging.eventgrid.EventGridPublisherAsyncClient#SendEventGridEvent
+    }
+
+    public void sendEvents() {
+        // BEGIN: com.azure.messaging.eventgrid.EventGridPublisherClient#CreateCloudEventClient
+        // Create a client to send events of CloudEvent schema (com.azure.core.models.CloudEvent)
+        EventGridPublisherClient<CloudEvent> cloudEventPublisherClient = new EventGridPublisherClientBuilder()
+            .endpoint(System.getenv("AZURE_EVENTGRID_CLOUDEVENT_ENDPOINT"))  // make sure it accepts CloudEvent
+            .credential(new AzureKeyCredential(System.getenv("AZURE_EVENTGRID_CLOUDEVENT_KEY")))
+            .buildCloudEventPublisherClient();
+        // END: com.azure.messaging.eventgrid.EventGridPublisherClient#CreateCloudEventClient
+
+        // BEGIN: com.azure.messaging.eventgrid.EventGridPublisherClient#SendCloudEvent
+        // Create a com.azure.models.CloudEvent.
+        User user = new User("Stephen", "James");
+        CloudEvent cloudEventDataObject = new CloudEvent("/cloudevents/example/source", "Example.EventType",
+            BinaryData.fromObject(user), CloudEventDataFormat.JSON, "application/json");
+
+        // Send a single CloudEvent
+        cloudEventPublisherClient.sendEvent(cloudEventDataObject);
+
+        // Send a list of CloudEvents to the EventGrid service altogether.
+        // This has better performance than sending one by one.
+        cloudEventPublisherClient.sendEvents(new ArrayList<CloudEvent>() {
+            {
+                add(cloudEventDataObject);
+                // add more CloudEvent objects
+            }
+        });
+        // END: com.azure.messaging.eventgrid.EventGridPublisherClient#SendCloudEvent
+
+        // BEGIN: com.azure.messaging.eventgrid.EventGridPublisherClient#CreateEventGridEventClient
+        // Create a client to send events of EventGridEvent schema
+        EventGridPublisherClient<EventGridEvent> eventGridEventPublisherClient = new EventGridPublisherClientBuilder()
+            .endpoint(System.getenv("AZURE_EVENTGRID_EVENT_ENDPOINT"))  // make sure it accepts EventGridEvent
+            .credential(new AzureKeyCredential(System.getenv("AZURE_EVENTGRID_EVENT_KEY")))
+            .buildEventGridEventPublisherClient();
+        // END: com.azure.messaging.eventgrid.EventGridPublisherClient#CreateEventGridEventClient
+
+        // BEGIN: com.azure.messaging.eventgrid.EventGridPublisherClient#SendEventGridEvent
+        // Create an EventGridEvent
+        User user2 = new User("John", "James");
```

Comment:
`user2` in javadoc of a separate method may not be ideal. Split this javadoc snippet into a new method and name this as `user` instead.